StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1203UnitTests.cs

```diff
@@ -63,6 +65,60 @@ public class Foo
             await this.VerifyCSharpDiagnosticAsync(testCode, firstDiagnostic, CancellationToken.None).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Verifies that the code fix will properly copy over the access modifier defined in another fragment of the partial element.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task TestCodeFixWithXmlDocumentationAsync()
+        {
+            var testCode = @"public class Foo
+{
+    private const string Before1 = ""test"";
+
+    public const string Before2 = ""test"";
+
+    private int field1;
+
+    private const string After1 = ""test"";
+
+    private int between;
+
+    public const string After2 = ""test"";
+}
+";
+
+            var fixedTestCode = @"public class Foo
```

Comment:
The block you're referring to is the before code, so the block that triggers the analyzer to throw a diagnostic, the block blow that is the fixed block.